ops/ansible/roles/bfd-server/tasks/install.yml

```diff
@@ -11,6 +11,14 @@
     shell: /bin/false
   become: true
 
+- name: Add server user to sudoers file
+  lineinfile:
+    path: /etc/sudoers
+    state: present
+    line: "{{ data_server_user }} ALL=(ALL) NOPASSWD: /sbin/iptables"
```

Comment:
Note that we may want to refactor this in the future to prevent the bfd fhir service account from being able to access the AWS metadata endpoint (and potentially getting secrets/credentials/sessions) by adding/removing iptables rules. 

Normally, AWS protects against remote users interacting with this endpoint, but you can do some crafty stuff with iptables, possibly leading to circumventing those protections.

It doesn't add or remove any protections currently in place, so it's fine for now. But I may want to refactor this in the future to implement better host based IDS/IPS/FW.